src/app.rs

```diff
@@ -203,7 +204,7 @@ impl App {
             &self.datadir,
             &self.address,
             self.port,
-            Net::TestNet,
+            self.net.clone(),
```

Comment:
I'm surprised that `Net` is actually not `Copy` but this is something we should probably raise with `golem-rpc-api` anyhow.